packages/system/rabbitmq-operator/Makefile

```diff
@@ -1,17 +1,10 @@
 NAME=rabbitmq-operator
-NAMESPACE=cozy-rabbitmq-operator
+NAMESPACE=cozy-$(NAME)
 
-show:
-	helm template --dry-run=server -n $(NAMESPACE) $(NAME) .
-
-apply:
-	helm upgrade -i -n $(NAMESPACE) $(NAME) .
-
-diff:
-	helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) .
+include ../../hack/package-system.mk
 
 update:
 	rm -rf templates/cluster-operator.yml
 	wget -O templates/cluster-operator.yml https://github.com/rabbitmq/cluster-operator/releases/latest/download/cluster-operator.yml
 	yq -i 'del(select(.kind=="Namespace"))' templates/cluster-operator.yml
-	sed -i 's/rabbitmq-system/cozy-rabbitmq-operator/g' templates/cluster-operator.yml
+	sed -i 's/rabbitmq-system/$(NAMESPACE)/g' templates/cluster-operator.yml
```

Comment:
I would rather keep update target without using variables, because commands here targets to concrete repositories, they are node depended on release name or namespace.

However I don't mind of replacing namespace with `$(NAMESPACE)` on line 10